.github/workflows/multibuild.yml

```diff
@@ -66,6 +71,10 @@ jobs:
         pip install virtualenv
     - name: Build and Install Wheels
       run: |
+        if [[ "$PLAT" == "arm64" ]]; then
+          sudo xcode-select -switch /Applications/Xcode_12.2.app
+          export SDKROOT=/Applications/Xcode_12.2.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.0.sdk
```

Comment:
Shouldn't we also `export MACOSX_DEPLOYMENT_TARGET=11.0` here?

Reply:
In the conda-forge config it's defined here:

https://github.com/conda-forge/openblas-feedstock/blob/master/.ci_support/osx_arm64_.yaml#L3

Reply:
Done in multibuild